# Entities Frame
Header header
sensor_msgs/CompressedImage img
TrackEntityMsg[] entities
